src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -86,15 +84,10 @@ def run_offline_reintegration_test(self, num_pool, data=False):
             output = self.dmg_command.pool_exclude(self.pool.uuid,
                                                    rank, t_string)
             self.log.info(output)
+            self.is_rebuild_done(3)
+            self.assert_on_rebuild_failure()
 
-            fail_count = 0
-            while fail_count <= 20:
-                pver_exclude = self.get_pool_version()
```

Comment:
Same as pver_exntend. If you remove this `pver_exclude`, doesn't it cause problem?

Reply:
Same as above. rebuild status checked for complete state and pool version read at the end.